hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/PrisonerSearchByBookingIdsResourceTest.kt

```diff
@@ -0,0 +1,103 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.search.resource
+
+import org.junit.jupiter.api.Test
+import org.springframework.web.reactive.function.BodyInserters
+import uk.gov.justice.digital.hmpps.prisonersearch.common.model.Prisoner
+import uk.gov.justice.digital.hmpps.prisonersearch.search.AbstractSearchDataIntegrationTest
+import uk.gov.justice.digital.hmpps.prisonersearch.search.model.Prisoner
+import uk.gov.justice.digital.hmpps.prisonersearch.search.model.nomis.dto.OffenderBooking
+import uk.gov.justice.digital.hmpps.prisonersearch.search.services.PrisonerListCriteria.BookingIds
+
+class PrisonerSearchByBookingIdsResourceTest : AbstractSearchDataIntegrationTest() {
+
+  override fun loadPrisonerData() {
+    val prisonerNumbers = List(12) { i -> "AN$i" }
+    var prisoners = ArrayList<Prisoner>()
+    prisonerNumbers.forEachIndexed { bookingId: Int, prisonNumber: String ->
+      val offenderBooking = getOffenderBooking(prisonNumber, bookingId.toLong())
+      val prisoner = Prisoner(offenderBooking, null, null)
+      prisoners.add(prisoner)
+    }
+    loadPrisoners(prisoners)
+  }
```

Comment:
This was the main change - to mimic how it was previously loading the data (via mocked out api calls)